carina-api/src/main/java/com/qaprosoft/apitools/validation/JsonValidator.java

```diff
@@ -41,6 +41,9 @@
 public class JsonValidator {
     private static final Logger LOGGER = Logger.getLogger(JsonValidator.class);
 
+	public JsonValidator() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ci.qaprosoft.com/sonarqube/coding_rules#rule_key=squid%3AS1186)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Hide this public constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ci.qaprosoft.com/sonarqube/coding_rules#rule_key=squid%3AS1118)